packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -133,6 +133,26 @@ export type CustomCallControlButtonPlacement =
   | 'afterParticipantsButton'
   | 'afterScreenShareButton';
 
+/* @conditional-compile-remove-from(stable): custom button injection */
+/**
+ * Placement for a custom button injected in the {@link CallControls} for
+ * MeetingCallControlBar sidepane buttons.
+ *
+ * 'first': Place the button on the left end (right end in rtl mode).
+ * 'afterChatButton': Place the button on the right (left in rtl mode) of the Chat button.
+ * ... and so on.
+ *
+ * It is an error to place the button in reference to another button that has
+ * been hidden via a {@link CallControlOptions} field.
+ *
+ * Multiple buttons placed in the same position are appended in order.
+ * E.g., if two buttons are placed 'first', they'll both appear on the left end (right end in rtl mode)
+ * in the order provided.
+ *
+ * @beta
+ */
+export type CustomMeetingControlButtonPlacement = 'first' | 'last' | 'afterChatButton';
+
```

Comment:
FYI: I did extract out the sidepane custom button placements here. @JamesBurnside and @prprabhu-ms.